frontend/src/components/table/DefaultFilter.tsx

```diff
@@ -0,0 +1,18 @@
+import { component$, PropFunction, Signal } from '@builder.io/qwik';
+
+export interface FilterInputProps {
+  filterValue: Signal<string>;
+  setFilter$: PropFunction<(val: string) => void>;
+}
+
+export const FilterInput = component$<FilterInputProps>(({ filterValue = '', setFilter$ }) => {
+  return (
+    <input
+      type="text"
+      value={(filterValue as Signal<string>).value}
+      onInput$={(ev: InputEvent) => setFilter$((ev.target as HTMLInputElement)?.value || '')}
+      placeholder={`Search...`}
+      style={{ width: '100%' }}
+    />
```

Comment:
Maybe we should consider style it with basic daisyUI?

Reply:
added some tailwind styles for input